flink-runtime/src/test/java/org/apache/flink/runtime/util/SerializedThrowableTest.java

```diff
@@ -63,20 +60,21 @@ public void testSerialization() {
             Class<?> clazz = userException.getClass();
 
             // check that we cannot simply copy the exception
-            try {
-                byte[] serialized = InstantiationUtil.serializeObject(userException);
-                InstantiationUtil.deserializeObject(serialized, getClass().getClassLoader());
-                fail("should fail with a class not found exception");
-            } catch (ClassNotFoundException e) {
-                // as we want it
-            }
+            assertThatThrownBy(
+                            () -> {
+                                byte[] serialized =
+                                        InstantiationUtil.serializeObject(userException);
+                                InstantiationUtil.deserializeObject(
+                                        serialized, getClass().getClassLoader());
```

Comment:
The first line should be moved out of this `assertThatThrownBy` if we expect `InstantiationUtil.deserializeObject(` throw exception.